src/Main.java

```diff
@@ -1,17 +1,18 @@
 import java.time.LocalDate;
+import java.util.ArrayList;
 import java.util.List;
 
 public class Main {
 
   public static void main(String[] args) {
 
-    Person a = new Person("John", LocalDate.parse("1969-03-15"), "+447984356766", "john@gmail.com");
-    Person b = new Person("Jane", LocalDate.parse("1998-04-09"), "+447220512328", "jane@gmail.com");
-    Person c = new Person("Harry", LocalDate.parse("1980-09-25"), "+447220012555", "harry@gmail.com");
-    Person d = new Person("Anne", LocalDate.parse("1969-03-15"), "+447220012222", "anne@gmail.com");
-    Person e = new Person("Jack", LocalDate.parse("1996-08-20"), "+447220012098", "jack@gmail.com");
+    List <Person> people = new ArrayList<>();
+    people.add(new Person("John", LocalDate.parse("1969-03-15"), "+447984356766", "john@gmail.com"));
+    people.add(new Person("Jane", LocalDate.parse("1998-04-09"), "+447220512328", "jane@gmail.com"));
+    people.add(new Person("Harry", LocalDate.parse("1980-09-25"), "+447220012555", "harry@gmail.com"));
+    people.add(new Person("Anne", LocalDate.parse("1969-03-15"), "+447220012222", "anne@gmail.com"));
+    people.add(new Person("Jack", LocalDate.parse("1996-08-20"), "+447220012098", "jack@gmail.com"));
 
-    List <Person> people = List.of(a,b,c,d,e);
```

Comment:
You could have kept the `List.of` method in place and combined it with the `new Person` constructor calls you also have.

Something like 
```
List.of(
new Person("blah"), 
new Person("blah2"));
```